src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectionHandler.ts

```diff
@@ -57,12 +61,6 @@ class ReconnectionProcess {
   async attemptPeriodicReconnection(options: ReconnectionOptions) {
     for (let i = 0; i < options.maxRetries; i++) {
       this.reconnectDisplay.update(i + 1);
-
-      const delayDuration = i == 0 && options.retryIntervalMilliseconds > ReconnectionProcess.MaximumFirstRetryInterval
-                            ? ReconnectionProcess.MaximumFirstRetryInterval
-                            : options.retryIntervalMilliseconds;
-      await this.delay(delayDuration);
```

Comment:
Since `retryIntervalMilliseconds` was put back to 20000ms, I think it's also necessary to bring back this logic.